bin/stacks

```diff
@@ -0,0 +1,333 @@
+#!/usr/bin/env python3
+
+from __future__ import print_function
+
+import os
+import sys
+import yaml
+import argparse
+import json
+import hashlib
+import time
+
+import boto.ec2
+import boto.vpc
+import boto.route53
+import boto.cloudformation
+import boto.s3
+import botocore.config
+import jinja2
+
+from boto.exception import BotoServerError
+from fnmatch import fnmatch
+from tabulate import tabulate
+
+
+# Resolve a basedir of this script
+if os.path.islink(__file__):
+    real_file = os.readlink(__file__)
+    basedir = os.path.realpath(os.path.dirname(real_file))
+else:
+    basedir = os.path.realpath(os.path.dirname(__file__))
+
+
+VERSION = '0.1'
+CONFIG_FILE = basedir + '/config.yaml'
+AWS_CONFIG_FILE = os.environ['HOME'] + '/.aws/credentials'
```

Comment:
Many times people work within a single region per account, so being able to specify a default region is quite useful.